eslint.config.js

```diff
@@ -8,6 +8,11 @@ export default [
 	{
 		files: ['*.js', '*.cjs'],
 	},
+	{
+		languageOptions: {
+			ecmaVersion: 2025,
```

Comment:
The default for this should be `latest` already, why add it explicitly now? Or is `latest` still on `2024`?

Reply:
Is fine for me but I wasn't sure if we want to use `latest`